targets/bornhack-2024-pov/badge_bsp_input.c

```diff
@@ -0,0 +1,113 @@
+// Board support package API: Generic stub implementation
+// SPDX-FileCopyrightText: 2024 Nicolai Electronics
```

Comment:
You are the author, replace and or add your name at your discretion. In this case I'd say just put your name on instead.